package.json

```diff
@@ -166,8 +163,10 @@
     "qrcode.react": "1.0.0",
     "qs": "6.5.3",
     "react": "17.0.2",
+    "react-17": "npm:react@17.0.2",
     "react-dfp": "^0.21.0",
     "react-dom": "17.0.2",
+    "react-dom-17": "npm:react-dom@17.0.2",
```

Comment:
**question (non-blocking):** What's the deal with the `npm:` syntax? That's how you declare an alias to a second version of the same lib?

Reply:
Yup! Its very useful, assuming you can remap the internals via some tool like jest